examples/playground/panel-lite-iframe.js

```diff
@@ -0,0 +1,159 @@
+const welcome=`\
+import panel as pn
+pn.extension()
+
+pn.pane.Markdown("Welcome! Try clicking the \`run\` button.").servable()
+`
+
+const htmlTemplate=`
+<!DOCTYPE html>
+<html>
+<head>
+	<title>My Page</title>
+    <script src="https://cdn.jsdelivr.net/pyodide/v0.24.1/full/pyodide.js"></script>
+    <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-3.3.0.min.js"></script>
+    <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-gl-3.3.0.min.js"></script>
+    <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-widgets-3.3.0.min.js"></script>
+    <script type="text/javascript" src="https://cdn.bokeh.org/bokeh/release/bokeh-tables-3.3.0.min.js"></script>
+    <script type="text/javascript" src="https://cdn.holoviz.org/panel/1.3.0/dist/panel.min.js"></script>
+</head>
+<body>
+    <div>I am a new template</div>
+	<div id="main"></div>
+    <div id="loading-spinner" style="height:100px;width:100px">Loading...</div>
+</body>
+</html>
+`
+
+// Fix Panel when running inside an iframe srcdoc
+// See https://github.com/holoviz/panel/issues/5506#issuecomment-1784433401
+const fixPanel = `
+from panel.io.location import Location
+Location.param.pathname.regex=r"^$|[\/]|srcdoc$"
+print("Panel Location issue fixed")
+`
+
+// See https://github.com/holoviz/panel/blob/8579e5cf322604e61b95bb1e10dcc57466298df1/panel/io/convert.py#L87
+const pyoidideScript = `
```

Comment:
We need to override the `PYODIDE_SCRIPT` to

- not reload pyodide, micropip and python packages
- listen to a message for the parent window to run some new code.